hl7apy/mllp.py

```diff
@@ -164,12 +164,12 @@ class MLLPServer(ThreadingMixIn, TCPServer):
     """
     allow_reuse_address = True
 
-    def __init__(self, host, port, handlers, timeout=10):
+    def __init__(self, host, port, handlers, timeout=10, RequestHandlerClass=MLLPRequestHandler):
```

Comment:
please use `lower_case_with_underscores` style for arguments names

Reply:
Sure. Happy to change this, but FWIW I wrote this to match the corresponding argument of `TCPServer`: https://docs.python.org/3/library/socketserver.html#socketserver.TCPServer